src/stores/storePosts.js

```diff
@@ -0,0 +1,45 @@
+import dispatcher from '../dispatcher';
+import {EventEmitter} from 'events';
+
+class StorePosts extends EventEmitter {
+  constructor() {
+    super();
+    this.posts = [];
+    this.addPost = this.addPost.bind(this);
+    this.getPosts = this.getPosts.bind(this);
+    this.change = this.change.bind(this);
+    this.handleActions = this.handleActions.bind(this);
+  }
+
+  addPost(post) {
+    this.posts = [post, ...this.posts];
+    console.log(this.posts);
+    this.change();
+  }
+
+  getPosts(posts) {
+    this.posts = [...this.posts, ...posts];
```

Comment:
```suggestion
    this.posts = posts;
```
вот здесь проблема в повышении количества постов с каждым переходом на страницу "Посты".